src/styles.css

```diff
@@ -181,4 +181,6 @@
 
 .table-deprecated mark {
     pointer-events: none;
-}
\ No newline at end of file
+}
+html, body { height: 100%; }
+body { margin: 0; font-family: Roboto, "Helvetica Neue", sans-serif; }
```

Comment:
We can't use roboto, please remove this line. We probably don't want the line above either...